carthage-archive.sh

```diff
@@ -0,0 +1,11 @@
+# Delete the old stuff
+rm -Rf Carthage
```

Comment:
Should we delete a MuxCore.framework.zip if one exists? Or does the zip command overwrite it?

Reply:
good point. checked the zip manual. it will overwrite every time.

`zip -u`, is the command to update/append